### DISCUSSION TITLE
creating custom subcollections from one subcollection

### DISCUSSION THREAD

**Discussion by robojiannis** [firecmsco/firecms#434] [Q&A]
I'm trying to utilise the propertyBuilder to show some fields of a subcollection at the root of the collection. Specifically:

I have a collection that is structured like this:
collectionA/documentA/subcollectionB/defaultDocument

the "defaultDocument" has a lot of fields, which I would like to show as properties of documentA, without the user having to click subcollectionB. Something along the lines of 

```
const visualProperties:PropertyBuilder = ({ values, entityId }: any) => ({
  title: "visuals",
  dataType: "array",
  of: {
    dataType: "reference",
    path: `newcards/${entityId}/superadmin`,
    previewProperties: ["image", "equipment", "renderedcards"]
  } 
});
```

but the path being the specific "defaultDocument" and the properties being the selected previewProperties.

Is there a way to go about that?

**Comment by fgatti675** [comment]
Hi @robojiannis I think there is no straightforward way to achieve this right now, feel to submit a ticket!
